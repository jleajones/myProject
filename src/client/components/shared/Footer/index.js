import React from 'react';
import Container from '@material-ui/core/Container';
import Typography from '@material-ui/core/Typography';

/**
 *
 * @returns {*}
 * @constructor
 */
const Footer = () => (
  <footer>
    <Container>
      <Typography>All Rights Reserved. Copyright &copy; 2019</Typography>
    </Container>
  </footer>
);

// const FooterWrapper = styled.footer`
//   padding: 12px 0;
// `;
export default Footer;
