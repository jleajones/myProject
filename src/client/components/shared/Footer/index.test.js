import React from 'react';
import { shallow } from 'enzyme';
import Footer from '.';

describe('<Footer />', () => {
  test('Renders component without crashing', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
