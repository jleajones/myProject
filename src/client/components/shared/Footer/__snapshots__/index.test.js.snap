// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders component without crashing 1`] = `
<footer>
  <WithStyles(ForwardRef(Container))>
    <WithStyles(ForwardRef(Typography))>
      All Rights Reserved. Copyright Â© 2019
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Container))>
</footer>
`;
